components/NewsItem.vue

```diff
@@ -1,18 +1,19 @@
 <template>
 	<!-- This compenent is used by data from pages api call as well as post api call -->
 	<article>
-		<figure class="preview-media">
+		<figure v-if="showPreviewImage" class="preview-media">
 			<img v-if="picture" :src="picture" :alt="altTag" class="news-item-img">
 			<WeCountLogo v-else class="news-item-img" />
 		</figure>
 		<h2 class="h3">
-			<nuxt-link :to="'/news-and-views/' + slug">
-				{{ title }}
-			</nuxt-link>
+			<a v-if="isExternalHref" :href="href" v-html="title" target="_blank" />
+			<nuxt-link v-else :to="href" v-html="title" />
 		</h2>
+		<div v-html="author" class="author" />
```

Comment:
If the link is to an external resource, this is misleading because it shows who posted the link to our site, not the actual author. I'd hide it if the link is to an external news item.